cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/passes/SymbolResolver.kt

```diff
@@ -188,6 +188,8 @@ open class SymbolResolver(ctx: TranslationContext) : ComponentPass(ctx) {
             // now).
             wouldResolveTo = scopeManager.resolveReference(current)
             if (wouldResolveTo !is VariableDeclaration && wouldResolveTo !is ParameterDeclaration) {
+                // TODO: not the best place for it and it is somewhat redundant
```

Comment:
I would like a small explanation here why we do need it to find better places for it in future refactoring.

Reply:
I already moved it to the "better place"